packages/pdfx/lib/src/viewer/pdf_page_image_provider.dart

```diff
@@ -1,8 +1,6 @@
-// ignore: unnecessary_import
 import 'dart:typed_data';
 import 'dart:ui' as ui show Codec;

```

Comment:
### This PR gets a different error:
`pdf_page_image_provider.dart:47:11: Error: 'ImmutableBuffer' isn't a type.`

### The fix
`import 'dart:ui' as ui show Codec, ImmutableBuffer;`